extensions/browser-bookmarks/src/index.tsx

```diff
@@ -126,6 +127,7 @@ export default function Command() {
   const hasVivaldi = browsers.includes(BROWSERS_BUNDLE_ID.vivaldi) ?? false;
   const hasZen = browsers.includes(BROWSERS_BUNDLE_ID.zen) ?? false;
   const hasWhale = browsers.includes(BROWSERS_BUNDLE_ID.whale) ?? false;
+  const hasLibreWolf = browsers.includes(BROWSERS_BUNDLE_ID.libreWolf) ?? false;
```

Comment:
Such a trivial thing, I admit, but this instance and many others in the patch add "libre" to a list that is otherwise lexicographically sorted. Would be nice for consistency to maintain that.